docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,181 @@
+
+✔️ DO include a short description for your package that describes what it's for.
+
+On the other hand, **Consider** recommendations should generally be followed, but there are legitimate exceptions to the rule:
+
+✔️ CONSIDER choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/en-us/nuget/reference/id-prefix-reservation).
+
+**Avoid** recommendations mention things that are generally not a good idea, but breaking the rule sometimes makes sense:
+
+❌ AVOID NuGet package references that demand an exact version.
+
+And finally, **Do not** recommendations indicate something you should almost never do:
+
+❌ DO NOT use the `LicenseUrl` metadata property.
+
+## Where do I define my package?
+
+TODO: Talk about how and where to make the suggested changes through out the guide (Visual Studio properties, project file, packages.config)
+
+## Framework targeting
+
+Authors should consider making their packages as inclusive as possible if they would like to maximize the number of potential consumers and overall impact in the .NET ecosystem.
+
+Highly inclusive packages are cross-platform and compatible with as many frameworks as possible. Today, that means using .NET Standard. .NET Standard is a specification of .NET APIs that are available on all .NET implementations.
+
+✔️ DO start with including a `netstandard2.0` target.
+> Most general-purpose libraries should not need APIs outside of .NET Standard 2.0. .NET Standard 2.0 is supported by all modern platforms and is the recommended way to support multiple platforms with one target.
+
+See the .NET [Cross-platform targeting guidance](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting) for more advanced suggestions regarding cross-platform and multitargeting scenarios.
+
+## Package metadata
+
+Metadata is a foundational component of any NuGet package. The quality of your metadata can vastly influence the discoverability and usability of your package.
+
+In Visual Studio, the recommended way to specify package metadata is to go Project > [Project Name] Properties > Package.
+
+Package metadata elements can also be specified directly in the project file (.csproj).
+
+Below is a table mapping and describing available packages metadata elements:
+
+| Visual Studio property name | csproj/MSBuild property name | Nuspec property name        | Description                                                                                                       |
+|-----------------------------|------------------------------|-----------------------------|-------------------------------------------------------------------------------------------------------------------|
+| `Package id`                | `PackageId`                  | `id`                        | The package identifier. A prefix from the identifier can be reserved if it meets the  criteria.                   |
```

Comment:
On the word "Criteria", please add a link to the criteria docs.  https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation

Reply:
FYI you have an extra space:

```suggestion
| `Package id`                | `PackageId`                  | `id`                        | The package identifier. A prefix from the identifier can be reserved if it meets the criteria.                   |
```